### DISCUSSION TITLE
Open for Discussions!

### DISCUSSION THREAD

**Discussion by poettering** [uapi-group/specifications#30] [Announcements]
First Post!

This is supposed to be a discussion forum for all things that don't really fit in the usual issue/PR scheme. Feel free to post anything related to UAPI specifications, documents, meetings and similar.

Please be civil!

**Comment by serebit** [comment]
Is the group open to representatives from desktop environment development teams such as KDE, GNOME, LXQT, XFCE, MATE, Budgie, etc?

**Reply by bluca** [reply]
Certainly - there's a few folks from GNOME OS already actively participating

**Reply by poettering** [reply]
And kde too actually (kinoite, in @travier)
